tests/opwen_email_server/utils/test_email_parser.py

```diff
@@ -289,3 +289,13 @@ def test_format_attachments_with_image(self):
 
         self.assertNotEqual(input_content, output_content)
         self.assertEqual(input_filename, output_filename)
+
+
+class DescendingCountTests(TestCase):
+    def test_descending_count_correct(self):
+        sent_at = '2020-02-01 21:09'
+        input_email = {'sent_at': sent_at}
```

Comment:
Nit: this wrapper object can be removed since `email_parser.descending_timestamp` takes string inputs instead of email object inputs.